.circleci/config.yml

```diff
@@ -103,7 +104,7 @@ commands:
           name: Configuring PATH
           command: |
             echo 'export PATH=~/venv-${PYTHON_VERSION_SHORT}/bin:~/.local/bin:${PATH}' >> ${BASH_ENV}
-            echo 'export RAIDEN_VERSION=$(python setup.py --version)' >> ${BASH_ENV}
+            echo 'export RAIDEN_VERSION=$(python3 setup.py --version)' >> ${BASH_ENV}
```

Comment:
Why do we explicitly need to use `python3` ? Or could we also use the current python version in use?

Reply:
In this [ci-run](https://app.circleci.com/pipelines/github/raiden-network/raiden/10616/workflows/5a0baf0f-d54c-4070-b6d8-292e57d6d50b/jobs/158596) using plain `python` causes a `python2` interpreter to be used during evaluation of `BASH_ENV`, causing a syntax error on the python3 syntaxed raiden `setup.py`.
This is most likely the cause of weird `pyenv` behaviour, since pyenv evaluates BASH_ENV before/during running the initialisation scripts `pyenv init`. [PEP 394](https://www.python.org/dev/peps/pep-0394/) can cause ambiguous behaviour of the `python` command, so that in this case it is symlinked to a `python2` version before pyenv can shim the executables. I am not entirely certain why this happens.
For the regular Linux part, apparently `python` and `python3` executables both symlink to the python 3 version. Either way, I would argue python3 is more specific anyways (PEP 394 again), so better than using plain `python`.